server/engine/src/main/resources/resourceBundles/messages_en.properties

```diff
@@ -108,3 +108,10 @@ jsonGenProcess.identifier2.groupItemError=%s group item must have unique data-na
 jsonGenProcess.identifier3.dataType=%s must be an integer data item defined without the symbol P in its picture string.
 jsonGenProcess.identifier6.dataType=%s should be elementary item with PIC X clause.
 jsonGenProcess.condition.dataType=% must be a level-88 item directly subordinate to %s.
```

Comment:
@zeibura Please help verify below messages

Reply:
the second one should be
%s must be an elementary item with a PIC X clause.

Reply:
updated